src/rzslider.js

```diff
@@ -817,9 +817,7 @@
       callOnChange: function() {
         if (this.options.onChange) {
           var self = this;
```

Comment:
Just to make it perfect, could you remove this line and call `onChange` directly on this? (When it's done, don't forget to rebase so there is one commit) ;)